geth/api/api.go

```diff
@@ -188,3 +193,35 @@ func (api *StatusAPI) JailCall(chatID, this, args string) string {
 func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
+
+// TODO: API package this stuff
+func (api *StatusAPI) Notify(token string) string {
+	log.Info("NOTIFY LOG", "token", token)
+
+	var NP fcm.NotificationPayload
+	NP.Title = "Status - new message"
+	NP.Body = "ping"
+
+	// TODO: Experiment with this
+	data := map[string]string{
+		"msg": "Hello World1",
+		"sum": "Happy Day",
+	}
+
+	ids := []string{
+		token,
+	}
+
+	c := fcm.NewFcmClient(serverKey)
+	c.NewFcmRegIdsMsg(ids, data)
+	c.SetNotificationPayload(&NP)
+	status, err := c.Send()
+
+	if err == nil {
```

Comment:
I would just do this way:

```go

    status, err := c.Send()
    if err != nil {
        log.Error("Notify failed:", err)
    }
```

Reply:
We might want to use this information (HTTP Status code) in future, but I got rid of it for now. Thanks